semgrep/semgrep/error.py

```diff
@@ -6,7 +16,7 @@ class SemgrepError(Exception):
     are displayed to the user.
     """
 
-    def __init__(self, *args: object, code: int = 1) -> None:
+    def __init__(self, *args: object, code: int = FATAL_EXIT_CODE) -> None:
```

Comment:
This was causing some tests to fail. `print_error_exit` was defaulting to `FATAL_EXIT_CODE`, while we were initially defaulting to `1`. This should clean it up :+1: